docs/cookbook/storageTank.js

```diff
@@ -0,0 +1,52 @@
+// tankDataX and tankDataY are X and Y-axis data for the last 24 hours,
+// obtained from elsewhere, e.g. const [tankDataX, tankDataY] = fetchData();
+// tankDataX is an array of Temporal.Absolute, and tankDataY is an array of numbers.
+
+// Show data starting from the most recent midnight in the tank's location (Stockholm)
+const tankTimeZone = Temporal.TimeZone.from('Europe/Stockholm');
+const tankMidnight = Temporal.now
+  .absolute()
+  .inTimeZone(tankTimeZone)
+  .getDate()
+  .withTime(Temporal.Time.from('00:00'))
+  .inTimeZone(tankTimeZone);
+const atOrAfterMidnight = (x) => Temporal.Absolute.compare(x, tankMidnight) >= 0;
+const dataStartIndex = tankDataX.findIndex(atOrAfterMidnight);
+const labelFormatter = new Intl.DateTimeFormat(undefined, {
+  weekday: 'short',
+  hour: 'numeric',
+  minute: 'numeric',
+  timeZone: Temporal.now.timeZone().name
```

Comment:
In general, you could just pass the timezone here, and toString would be called, giving the right result, right?

Reply:
Yes, that works too — I'll add a fixup in the next cookbook pull request.